getting-started/ci-setup/gitlab/custom-docker-runners.md

```
@@ -30,3 +30,19 @@ COMMIT_INFO_REMOTE=$CI_REPOSITORY_URL
 ```
 
 Adding these environment variables to your `.yml` file will allow your custom docker runner to report the correct information to Currents and have all the available information in the dashboard to use it as a usual Gitlab CI/CD tests execution.
+
+## Fallback: Using CURRENTS_CI_URL
+
+If your GitLab environment variables are not forwarded to the Docker container and auto-detection fails, you can explicitly provide the CI URL using the `CURRENTS_CI_URL` environment variable:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)